webui/react/src/components/Dropdown.tsx

```diff
@@ -122,10 +122,16 @@ const Dropdown: React.FC<Props> = ({
 
     event.stopPropagation();
 
+    const isAntPicker = (
+      typeof event.target.className === 'string'
+      && event.target.className.indexOf('ant-picker') >= 0
+    );
```

Comment:
**chore:** prevent dismissal of the dropdown when the user clicks `ok` on the date picker or outside of the date picker (should only dismiss the date picker and go back to the filters dropdown)

Reply:
This won't handle the outside of the date picker but that might ok unless you have a better idea:
```
    const isAntPicker = event.target.closest('.ant-picker');
    const isAntDropdownPicker = event.target.closest('.ant-picker-dropdown');

    if (isAntPicker || isAntDropdownPicker) {
      return;
    }
```

Reply:
Changed to event.target.closest('div').className.indexOf('ant-picker') >= 0